docs/cloud/features/observability/development_environment.md

```diff
@@ -0,0 +1,45 @@
+# Development Environment
+
+The Tobiko Cloud Environments page provides an overview of all the environments that exist in your project.
+
+![tcloud environment page](./development_environment/environments.png)
+
+The page's table includes a link to each environment's page, along with the environment's creation date, the date it was last updated, and the date it will expire if not updated again.
+
+## Environment page
+
+Clicking an environment's name in the table takes you to its environment page.
+
+The page begins with a at-a-glance summary of the most recent plan applied to the environment, including:
```

Comment:
Nit: should it be `with an at-a-glance`?

Reply:
resolved in [38a60d6](https://github.com/TobikoData/sqlmesh/pull/3515/commits/38a60d6976916791f5eb7520caaf83f6839ddd20)